### DISCUSSION TITLE
data from other species (question)

### DISCUSSION THREAD

**Discussion by silvia1234567890** [milaboratory/mixcr#1403] [Q&A] [answered]
Hello,

Could I use MiXCR to process data from species other than Human, Mice, Rat, Spalax, Alpaca, Monkey? For example, trout or turbot data?

Thank you

**Comment by mizraelson** [comment, marked as answer]
Hi,

MiXCR should work with any species but requires the correct reference. [This article](https://mixcr.com/mixcr/guides/create-custom-library/#de-novo-libraries) will guide you through the process of creating a reference for non-model organisms, provided you have FASTA files with all gene segments.

Sincerely,
Mark

**Comment by silvia1234567890** [comment]
Great, thank you so much.

Just one question, the headers of the gene segments need to be in a specific format?

I checked that link and in the example they are:
```
>IGHV12-348
GATGCTGGAGTTATCCAGTCA...
```
```
>IGHJJ1-528
TAACAACCAGGCCCAG...
```

And I have fasta files like this:
```
>aOncmyk_Chr8_CM023226.2_TRAV4-6_F_61345213_61345541
GTGACAGTCTTCAGAC...
```
```
>aOncmyk_Chr8_NC_048572_RS_TRAJ1_F_61944197_61944292
AGTGACTGGAACAGGA...
```

Thank you

**Comment by silvia1234567890** [comment]
Please, I need a confirmation if "yes" or "no" and I will close this issue, thank you!

**Comment by mizraelson** [comment]
Hi,

You can use an arbitrary header, but keep in mind that it will be used as a gene name in the output tables. If you choose to use '|' as a separator, only the first element will be assigned as a name. For example: '> TRBV14-3|homosapiens|SRR0309337'.

Sincerely,
Mark

**Comment by silvia1234567890** [comment]
Oh ok, thank you so much for your help!!
